jme3-core/src/main/java/com/jme3/anim/AnimComposer.java

```diff
@@ -546,6 +552,9 @@ public void read(JmeImporter im) throws IOException {
         animClipMap = (Map<String, AnimClip>) ic.readStringSavableMap("animClipMap", new HashMap<String, AnimClip>());
         globalSpeed = ic.readFloat("globalSpeed", 1f);
         layers = (Map<String, AnimLayer>) ic.readStringSavableMap("layers", new HashMap<String, AnimLayer>());
+        if (!layers.containsKey(DEFAULT_LAYER)) {
```

Comment:
Please use `map.putIfAbsent(key, value)` instead. This is generally considered more concise and potentially more efficient than the manual check.

eg:
```java
map.putIfAbsent(DEFAULT_LAYER, new AnimLayer(DEFAULT_LAYER, null);
```